#! /bin/sh /usr/share/dpatch/dpatch-run
## myplugin_includes.dpatch by Sebastian Harl <sh@tokkee.org>
##
## DP: Let the example plugin include the core headers from
## DP: /usr/include/collectd/core.

@DPATCH@

diff a/contrib/examples/myplugin.c b/contrib/examples/myplugin.c
--- a/contrib/examples/myplugin.c
+++ b/contrib/examples/myplugin.c
@@ -38,9 +38,9 @@
 
 #endif /* ! HAVE_CONFIG */
 
-#include <collectd/collectd.h>
-#include <collectd/common.h>
-#include <collectd/plugin.h>
+#include <collectd/core/collectd.h>
+#include <collectd/core/common.h>
+#include <collectd/core/plugin.h>
 
 /*
  * data source definition:
